src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,122 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+
+    private List<Node<K, V>>[] buckets;
```

Comment:
It should be list of nodes that will create unidirectional linked list
```suggestion
    private Node<K, V>[] buckets;
```